doc/README.yaml.md

```diff
@@ -0,0 +1,240 @@
+
+# README for Open-CE build YAML files
+
+
+This README documents the YAML files used by the build scripts
+[build_env.py](https://github.com/open-ce/open-ce/blob/master/doc/README.build_env.md)
```

Comment:
Links should be relative instead of picking a particular branch.

Reply:
Done (here and universally in the file), for all cases within the open-ce repo itself.  (Exception is URLs in the file that point to a separate feedstock repository not within open-ce, which are general and not branch-specific.)